### ISSUE TITLE
User config prefixed by Path.home() breaks in AWS Lambda or any system without /home

### ISSUE THREAD

**Issue by joaodiogocosta** [ultralytics/yolov5#4718]
## 🐛 Bug

This commit 8e94bf62d9aa588982daec58e89dd2bb682a1f0e forces the user config dir to be inside /home/..., which is not compatible with AWS Lambda (only /tmp is writable).

I believe this should be configurable somehow so that the new user config dir can be set by the developer.

```
FileNotFoundError: [Errno 2] No such file or directory: '/home/sbx_user1051/.config/Ultralytics'
  File "lambda_function.py", line 44, in process_frames
    processing_context = FindObjects(processing_context).run()
  File "pyworker/models/mod/find_objects.py", line 15, in __init__
    self.model = object_detector()
  File "pyworker/models/ml_models.py", line 5, in object_detector
    return torch.hub.load('ultralytics/yolov5', 'custom', path=model_path)
  File "torch/hub.py", line 364, in load
    model = _load_local(repo_or_dir, model, *args, **kwargs)
  File "torch/hub.py", line 393, in _load_local
    model = entry(*args, **kwargs)
  File "/tmp/hub/ultralytics_yolov5_master/hubconf.py", line 70, in custom
    return _create(path, autoshape=autoshape, verbose=verbose, device=device)
  File "/tmp/hub/ultralytics_yolov5_master/hubconf.py", line 30, in _create
    from models.yolo import Model
  File "models/yolo.py", line 17, in <module>
    from models.common import *
  File "models/common.py", line 23, in <module>
    from utils.plots import Annotator, colors
  File "utils/plots.py", line 23, in <module>
    CONFIG_DIR = user_config_dir()  # Ultralytics settings dir
  File "utils/general.py", line 111, in user_config_dir
    path.mkdir()  # make dir if required
  File "pathlib.py", line 1288, in mkdir
    self._accessor.mkdir(self, mode)
```

## Environment

- AWS Lambda (Python)

**Comment by glenn-jocher** [comment]
@joaodiogocosta fixed in PRs #4726 and #4727, closing issue. 

Thank you for your contributions and let us know if you run into any other issues!

**Comment by joaodiogocosta** [comment]
Gotcha ✌️
